test/src/test/java/hudson/diagnosis/OldDataMonitorTest.java

```diff
@@ -88,6 +88,7 @@
     }
 
     @Issue("JENKINS-19544")
+    @Ignore
```

Comment:
I think a `FIXME` should be added here explaining why it was removed, then this PR is ready to be merged IMO.